astropy/units/tests/test_logarithmic.py

```diff
@@ -77,18 +77,20 @@ def test_subclass_creation(self, lu_cls, physical_unit):
 
 
 def test_predefined_magnitudes():
+    # these magnitudes are only approximations to the true definition
     assert_quantity_allclose((-21.1*u.STmag).physical,
-                             1.*u.erg/u.cm**2/u.s/u.AA)
+                             1.*u.erg/u.cm**2/u.s/u.AA, rtol=.1)
     assert_quantity_allclose((-48.6*u.ABmag).physical,
-                             1.*u.erg/u.cm**2/u.s/u.Hz)
+                             1.*u.erg/u.cm**2/u.s/u.Hz, rtol=.1)
+
     assert_quantity_allclose((0*u.M_bol).physical, c.L_bol0)
     assert_quantity_allclose((0*u.m_bol).physical,
                              c.L_bol0/(4.*np.pi*(10.*c.pc)**2))
 
 
 def test_predefined_reinitialisation():
-    assert u.mag('ST') == u.STmag
```

Comment:
As noted above, I think we have to continue to have this work to keep @pllim happy!

Reply:
Can we keep the `assert` so that we don't regress? and maybe add a note.

Reply:
Oops, ignore that.